src/app/legacy/containers/PageHandlers/withHashChangeHandler/index.test.jsx

```diff
@@ -10,14 +10,48 @@ const Fixture = withHashChangeHandler(() => (
   </>
 ));
 
+const CyrillicFixture = withHashChangeHandler(() => (
```

Comment:
Should we also check if Hindi is OK (given that we're planning to use this in the JumpTo component for hindi?